pkgs/by-name/bu/bugwarrior/package.nix

```
@@ -0,0 +1,97 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  taskwarrior3,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "bugwarrior";
+  version = "2.0.0";
+  disabled = python3.pythonOlder "3.10";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "GothenburgBitFactory";
+    repo = pname;
```

```suggestion
    repo = "bugwarrior";
```
https://github.com/NixOS/nixpkgs/issues/277994